gaugan/models/gaugan.py

```diff
@@ -153,7 +154,7 @@ def train_generator(
 
         gradients = tape.gradient(total_loss, self.combined_model.trainable_variables)
```

Comment:
add + self.encoder.trainable_variables to tape.gradient

Reply:
https://github.com/soumik12345/tf2_gans/issues/14#issuecomment-1034590292